cmd/uplink/cmd/setup.go

```diff
@@ -120,7 +120,11 @@ func cmdSetupInteractive(cmd *cobra.Command, setupDir string) error {
 		return Error.Wrap(err)
 	}
 
-	uplink, err := libuplink.NewUplink(ctx, nil)
+	uplinkCfg := libuplink.Config{}
+	uplinkCfg.Volatile.TLS.PeerCAWhitelistPath = vip.GetString("tls.peer-ca-whitelist-path")
+	uplinkCfg.Volatile.TLS.SkipPeerCAWhitelist = !vip.GetBool("tls.use-peer-ca-whitelist")
```

Comment:
Are there other arguments that should be supplied as well, not just TLS?

Doesn't gateway have the same issue?

Reply:
Looks like the gateway already does this: https://github.com/storj/storj/blob/master/cmd/gateway/main.go#L244

Reply:
And yea, there are other uplink config fields that aren't being set but I don't think the existing bound config includes the others.

Reply:
Note that we have this helper method: https://github.com/storj/storj/blob/master/cmd/uplink/cmd/root.go#L77-L91

Reply:
@kaloyan-raev that's cool, I was not aware. We'd have reorganize the config structs i think to use that though.